doc/main/tbb_userguide/std_invoke.rst

```diff
@@ -0,0 +1,237 @@
+.. _std_invoke:
+
+Invoke a Callable Object
+==========================
+
+Starting from C++17, the requirements for callable objects passed to algorithms or Flow Graph nodes are relaxed. It allows using additional types of bodies. 
+Previously, the body of the algorithm or Flow Graph node needed to be a Function Object (see `C++ Standard Function Object <https://en.cppreference.com/w/cpp/utility/functional>`_) and provide an 
+``operator()`` that accepts input parameters. 
+
+Now the body needs to meet the more relaxed requirements of being Callable (see `C++ Standard Callable <https://en.cppreference.com/w/cpp/named_req/Callable>`_) that covers three types of objects:
+
+* **Function Objects that provide operator(arg1, arg2, ...)**, which accepts the input parameters
+* **Pointers to member functions** that you can use as the body of the algorithm or parallel construct
```

Comment:
For me, it is not clear what "parallel construct" is in this context? May be we can use Flow Graph node directly instead?